portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,8 @@ function addRandomGreeting() {
 }
 
 function getGreeting() {
-  fetch('/data').then(response => response.text()).then((text) => {
-    document.getElementById('data-greeting').innerText = text;
+  fetch('/data').then(response => response.json()).then((outputStrings) => {
```

Comment:
I would use greeting instead of outputStrings.

Reply:
Done